packages/web/src/components/element/TeeCard.vue

```diff
@@ -0,0 +1,107 @@
+<template>
+  <div
+    class="fr-grid-row"
+    :class="linkClasses"
+  >
+    <div
+      v-if="imglink"
+      class="image-container fr-col-12 fr-col-md-3 fr-mb-2v fr-mb-md-0"
+    >
+      <img
+        :src="imglink"
+        alt=""
+        class="image"
+      />
+    </div>
+    <div class="content fr-col-12">
+      <h1 v-if="title">{{ title }}</h1>
+      <p><slot></slot></p>
+      <TeeButtonExternalLink
+        v-if="link"
+        :href="link"
+        class="fr-mt-3v"
+      >
+        {{ linkText }}
+      </TeeButtonExternalLink>
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { defineProps, withDefaults } from 'vue'
+import { CardType } from '@/types/elementsPropsTypes'
+
+interface Props {
+  type?: CardType
+  title?: string | number | undefined
+  link?: string | undefined
+  linkText?: string | undefined
+  imglink?: string | undefined
+}
+
+const props = withDefaults(defineProps<Props>(), {
+  type: CardType.Default,
+  title: undefined,
+  link: undefined,
+  linkText: undefined,
+  imglink: undefined
```

Comment:
note: par défaut, c'est déjà `undefined`

Reply:
Pour info, j'ai laissé parce que j'ai un warning eslint : Prop 'link' requires default value to be set.eslint[vue/require-default-prop](https://eslint.vuejs.org/rules/require-default-prop.html)